### PR TITLE
Edits

### PR DESCRIPTION
**Pull request by mmimac**
A batch of edits for documentation. :)

### CODE REVIEW

File: src/lib/docs/SessionKit/LoginResult.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
 ---
 title: LoginResult
-description: The data returned from a call to the login method from the Session Kit
-category: SessionKit
+description: The data returned from a call to the login method from the Session Kit.
+category: Session Kit
```
Comment (aaroncox) [inline src/lib/docs/SessionKit/LoginResult.md:4]: We cannot have spaces in these `category` names, it breaks the URL generation for some reason.

File: src/lib/docs/SessionKit/LoginResult.md [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
 published: true
 hidden: true
-requiresReview: true
 ---
 
 # LoginResult
 
-Upon a successful call to the [Login](/docs/sessionkit/login) method of the [SessionKit](/docs/sessionkit/session-kit-factory), the response returned will be an object that matches the `LoginResult` interface.
+Upon a successful call to the [Login](/docs/sessionkit/login) method of the [Session Kit](/docs/sessionkit/session-kit-factory), the response returned will be an object that matches the `LoginResult` interface.
```
Comment (aaroncox) [inline src/lib/docs/SessionKit/LoginResult.md:11]: I've been intentionally doing 2 things here: 

- Using `SessionKit` when referring to the "code object" (you can see the url goes to the "session-kit-factory" page)
- Using `Session Kit` when it refers to the overall product (and the url goes to the landing page of /docs/sessionkit) 

A lot of these instances that were changed in this PR break that pattern and before we merge any of this in, I want to just make a decision if we're changing that everywhere.

**Review by aaroncox — COMMENTED** [review]
Thanks for these! I left a couple comments before we merge things in and get too much further.

**Comment by mmimac** [comment]
Everything's edited now, and I think all the SessionKit spacing has been corrected too. :) The only ones that need your attention should be:

- WalletPlugin.md, which has an incomplete section: _If the `WalletPlugin` or external application modifies the transaction and returns it, it may invalidate any signatures previously created by the [TransactPlugin](/docs/sessionkit/plugin-transact) calls that were originally made. We would recommend that the wallet should not_
- SessionKit/SessionKit.md, which looks like it's still being drafted
- LoginPlugin.md, which says _The login lifecycle currently has 3 points at which hooks can be established,_ but there are only 2 points listed.

**Pull request merged**